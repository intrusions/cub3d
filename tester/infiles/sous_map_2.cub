NO ./texture/no.xpm
SO ./texture/so.xpm
WE ./texture/we.xpm
F  10,19,0



1111111111111111111111111111111
100000000000000000S000111111
1111111111111111111111111111111
EA ./texture/ea.xpm
C 225,30,0
100000000000000000000000000000111111111
11111111111111111N00000111111111111
1000000000000000000000000000000001
1000000000000000000000000000000001
1000000000000000000000000000000001
100000000000000000N000000000000001
1000000000000000000000000000000001
100000000S000000000000000000000001
11111111111111111111111111111111111111111111111