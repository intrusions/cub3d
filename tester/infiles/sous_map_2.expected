Loading scene...
Loading map...
Error
	sous_map_2.cub: Map: is open
	sous_map_2.cub: Map: too many players
Quitting...
